### DISCUSSION TITLE
Failed to bind to the CLR. Error code 0x8007007E

### DISCUSSION THREAD

**Discussion by cubt85iz** [wixtoolset/issues#7613] [Questions]
Hey all,

We're experiencing a new issue when running our installations in a Windows Sandbox. It appears that we're experiencing a failure when attempting to execute a custom action and the error displayed in the title is shown in our log file. We've been able to workaround the issue temporarily by uninstalling KB5011048.

VS: 2022 Enterprise
OS: Win 10 x64 (all updates as of yesterday)
.NET: CA projects are targetting 4.7.2

Haven't tried to reproduce on small scale, but our project consists of a bundle, multiple MSIs and a couple of CA projects. Let me know if there's any additional information I can provide.

**Comment by cubt85iz** [comment]
Posted sample project here: https://github.com/cubt85iz/SampleInstallation

**Comment by BMurri** [comment]
From the outside looks like it might be related to https://developercommunity.visualstudio.com/t/KB5011048-breaks-PowerShell-in-Windows-S/10187046

**Comment by cubt85iz** [comment]
Are you suggesting that invoking the sandbox from a Powershell script is causing the issue?
